pages/edit.js

```diff
@@ -0,0 +1,335 @@
+                      event,
+                      newTimer.id,
+                      newTimerChain,
+                      setNewTimerChain
+                    )
+                  }
+                />
+                <Label htmlFor="exerciseName"> Exercise</Label>
+              </InputAndLabelWrapper>
+              <InputAndLabelWrapper>
+                <Select
+                  id="minutes"
+                  value={newTimer.minutes}
+                  onChange={(event) =>
+                    changeTimerChain(
+                      event,
+                      newTimer.id,
+                      newTimerChain,
+                      setNewTimerChain
+                    )
+                  }
+                >
+                  {timeOptions.map((option) => {
+                    return (
+                      <option key={option} value={option}>
+                        {option}
+                      </option>
+                    );
+                  })}
+                </Select>
+                <Label htmlFor="minutes">min.</Label>
+              </InputAndLabelWrapper>
+              <span>:</span>
+              <InputAndLabelWrapper>
+                <Select
+                  id="seconds"
+                  value={newTimer.seconds}
+                  onChange={(event) =>
+                    changeTimerChain(
+                      event,
+                      newTimer.id,
+                      newTimerChain,
+                      setNewTimerChain
+                    )
+                  }
+                >
+                  {timeOptions.map((option) => {
+                    return (
+                      <option key={option} value={option}>
+                        {option < 10 ? `0${option}` : option}
+                      </option>
+                    );
+                  })}
+                </Select>
+                <Label htmlFor="seconds"> sec.</Label>
+              </InputAndLabelWrapper>
+            </Li>
+          ))}
+        </FormList>
+        <ButtonWrapper>
+          <AddButton
+            type="button"
+            onClick={() => handleAdd(newTimerChain, setNewTimerChain)}
+          >
+            <Image alt="add" src={add} width="64" height="64" />
+          </AddButton>
+          <DeleteButton
+            type="button"
+            onClick={() => handleRemove(newTimerChain, setNewTimerChain)}
+          >
+            <Image alt="remove" src={remove} width="64" height="64" />
+          </DeleteButton>
+        </ButtonWrapper>
+        <ApplyWrapper>
+          <DummyDiv />
+          <Apply type="submit">
+            <Image alt="hook" src={hook} width="42" height={41.71} />
+            <div>Apply</div>
+          </Apply>
+        </ApplyWrapper>
+      </Form>
+    </>
+  );
+}
+
+const HeadlineWrapper = styled.div`
+  position: fixed;
+  width: 100%;
+  backdrop-filter: blur(10px);
+`;
+
+const TopGap = styled.div`
+  height: 75px;
+`;
```

Comment:
Using TopGap styled component instead of giving the Form or the headlineWrapper the nessecary styling.
I think you can cut this out and just give a "margin-top" or "margin-bottom" to one of the existing components